src/application/pokegear/configure/overlay_101_021EED44.c

```diff
@@ -0,0 +1,387 @@
+     .y = 0,
+     .z = 0,
+     .animation = 0,
+     .drawPriority = 1,
+     .pal = 6,
+     .vram = NNS_G2D_VRAM_TYPE_2DMAIN,
+     .paletteMode = 0,
+     .unk_1C = 0,
+     .unk_20 = 0,
+     .unk_24 = 0,
+     },
+    {
+     .resourceSet = 0,
+     .x = 0,
+     .y = 0,
+     .z = 0,
+     .animation = 1,
+     .drawPriority = 1,
+     .pal = 6,
+     .vram = NNS_G2D_VRAM_TYPE_2DMAIN,
+     .paletteMode = 0,
+     .unk_1C = 0,
+     .unk_20 = 0,
+     .unk_24 = 0,
+     },
+    {
+     .resourceSet = 0,
+     .x = 0,
+     .y = 0,
+     .z = 0,
+     .animation = 2,
+     .drawPriority = 1,
+     .pal = 6,
+     .vram = NNS_G2D_VRAM_TYPE_2DMAIN,
+     .paletteMode = 0,
+     .unk_1C = 0,
+     .unk_20 = 0,
+     .unk_24 = 0,
+     },
+    {
+     .resourceSet = 0,
+     .x = 0,
+     .y = 0,
+     .z = 0,
+     .animation = 3,
+     .drawPriority = 1,
+     .pal = 6,
+     .vram = NNS_G2D_VRAM_TYPE_2DMAIN,
+     .paletteMode = 0,
+     .unk_1C = 0,
+     .unk_20 = 0,
+     .unk_24 = 0,
+     },
+    {
+     .resourceSet = 0,
+     .x = 0,
+     .y = 0,
+     .z = 0,
+     .animation = 8,
+     .drawPriority = 1,
+     .pal = 4,
+     .vram = NNS_G2D_VRAM_TYPE_2DMAIN,
+     .paletteMode = 0,
+     .unk_1C = 0,
+     .unk_20 = 0,
+     .unk_24 = 0,
+     },
+};
+
+static const PokegearCursorGrid sCursorButtons[6] = {
+    { 0, 2, 1, 3, 3, 48,  44,  -22, 22, -18, 18 },
+    { 1, 0, 2, 4, 4, 128, 44,  -22, 22, -18, 18 },
+    { 2, 1, 0, 5, 5, 208, 44,  -22, 22, -18, 18 },
+    { 3, 5, 4, 0, 0, 48,  116, -22, 22, -18, 18 },
+    { 4, 3, 5, 1, 1, 128, 116, -22, 22, -18, 18 },
+    { 5, 4, 3, 2, 2, 208, 116, -22, 22, -18, 18 },
+};
+
+BOOL PokegearConfigure_LoadGFX(PokegearConfigureAppData *configureApp) {
+    switch (configureApp->substate) {
+    case 0:
+        PokegearConfigure_InitBGs(configureApp);
+        PokegearConfigure_LoadGraphics(configureApp);
+        PokegearConfigure_CreateSpriteManager(configureApp);
+        PokegearConfigure_LoadPalettes(configureApp, TRUE);
+        break;
+    case 1:
+        PokegearConfigure_CreateSprites(configureApp);
+        PokegearConfigure_CreateCursor(configureApp);
+        PokegearConfigure_LoadContextMenuText(configureApp);
+        PokegearConfigure_SetInitialCursorState(configureApp);
+        configureApp->substate = 0;
+        return TRUE;
+    }
+    ++configureApp->substate;
+    return FALSE;
+}
+
+BOOL PokegearConfigure_UnloadGFX(PokegearConfigureAppData *configureApp) {
+    PokegearConfigure_UnloadContextMenuText(configureApp);
+    PokegearConfigure_RemoveCursor(configureApp);
+    PokegearConfigure_DeleteSprites(configureApp);
+    PokegearConfigure_DestroySpriteManager(configureApp);
+    PokegearConfigure_UnloadGraphics(configureApp);
+    PokegearConfigure_UnloadBGs(configureApp);
+    return TRUE;
+}
+
+int PokegearConfigure_ContextMenu(PokegearConfigureAppData *configureApp) {
+    int input;
+
+    input = TouchscreenListMenu_HandleInput(configureApp->contextMenu);
```

Comment:
merge declaration and assignment?

Reply:
I'm not committed enough to block a merge over this but I reserve the right for anyone in the future to go back and change it